### ISSUE TITLE
Create a "who's using sapper" section

### ISSUE THREAD

**Issue by stephane-vanraes** [sveltejs/sapper#1113]
Should we make a section on the homepage for "who's using Sapper" ?
Similar to the one on https://svelte.dev/ ?

This would point people that are interested in using Sapper to actual real world examples.

**Comment by antony** [comment]
I think we'd be open to this. Do you fancy creating a PR for said page?

**Comment by kevmodrome** [comment]
You could make a copy of the who's using svelte file in the community repo to use with it, if that makes sense.

**Comment by TheComputerM** [comment]
Can I add https://svelte-materialify.vercel.app/ to this section? This is not a company but a framework.

**Comment by kevmodrome** [comment]
I don't think that is a good fit @TheComputerM. Imagine what would happen if it was merged. Any and all libraries/frameworks would need to be added.

**Comment by TheComputerM** [comment]
I understand, but just to be clear I was focused on the website as it is made in sapper and is pretty large and complex to be included in a showcase.
